amazon_transcribe/client.py

```diff
@@ -140,6 +141,8 @@ async def start_stream_transcription(
             results. A higher stability level means that the transcription results
             are less likely to change. Higher stability levels can come with lower
             overall transcription accuracy. Defaults to "high" if not set explicitly.
+        :param language_model_name
```

Comment:
It looks like this is missing a close `:`, would you mind adding that?

```suggestion
        :param language_model_name:
```